superpowered/Superpowered.d.ts

```diff
@@ -520,6 +520,7 @@ export declare class Superpowered {
     createFloatArray(length: number): SuperpoweredFloat32Buffer;
     bufferToWASM(output: SuperpoweredFloat32Buffer, input: AudioBuffer): void;
     bufferToJS(input: SuperpoweredFloat32Buffer, output: AudioBuffer): void;
+    createViewFromType(type: number, pointer: SuperpoweredMemoryPointer, length: number): SuperpoweredFloat32Buffer;
```

Comment:
`type` will be called with `9` to indicate `Float32`. I couldn't figure out how to make that a named constant to be called from our side (I tried an enum as well as simply a number constant in this file). I'd appreciate a suggestion here as I don't like magic numbers even though it is used within `SuperpoweredGlue.js`

Reply:
If we can figure that out, I'll update the RC version afterwards.

Reply:
You should be able to use a `const enum` https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

```ts
export declare const enum SampleType {
    Uint8 = 1,
    Int8 = 2,
    Uint16 = 3,
    Int16 = 4,
    Uint32 = 5,
    Int32 = 6,
    BigUint64 = 7,
    BigInt64 = 8,
    Float32 = 9,
    Float64 = 10,
}
```

(technically you could write this only specifying the first value, because of TypeScript's auto-incrementing enum behavior, but that seems unnecessarily cryptic:

```ts
export declare const enum SampleType {
    Uint8 = 1,
    Int8,
    Uint16,
    Int16,
    Uint32,
    Int32,
    BigUint64,
    BigInt64 ,
    Float32,
    Float64,
}
```